utils/xscmanager.go

```diff
@@ -19,3 +26,33 @@ func CreateXscServiceManager(serviceDetails *config.ServerDetails) (*xsc.XscServ
 	}
 	return xsc.New(serviceConfig)
 }
+
+func SendXscLogMessageIfEnabled(errorLog *services.ExternalErrorLog, xscManager *xsc.XscServicesManager) error {
+	if !reportLogErrorEventPossible(xscManager) {
+		return nil
+	}
+	err := xscManager.SendXscLogErrorRequest(errorLog)
+	if err == nil {
+		log.Info("Error successfully reported to Coralogix")
+	}
+	return err
+}
```

Comment:
```suggestion
	if !reportLogErrorEventPossible(xscManager) {
		return nil
	}
	return xscManager.SendXscLogErrorRequest(errorLog)
}
```
No need to mention that reporting logs are done without issues, it doesn't matter to the user, specially not in `info` level. if we will get an error we will see that at the logs.